piq/brisque.py

```diff
@@ -191,7 +191,7 @@ def _natural_scene_statistics(luma: torch.Tensor, kernel_size: int = 7, sigma: f
     std = ((std - mu_sq).abs().sqrt())
 
     luma_nrmlzd = (luma - mu) / (std + C)
-    features = []
+    features: List = []
```

Comment:
I think this one is a bit redundant since it can be nothing then a list. 
Would you consider removing this type annotation?

Reply:
Yes. 
I've experimented with inline typing and forgot to remove this. Anything else?